tools/rosgraph/src/rosgraph/impl/graph.py

```diff
@@ -200,9 +203,9 @@ def __init__(self, start, end, label=''):
         self.start = start
         self.end = end
         self.label = label
-        self.key = "%s|%s"%(self.start, self.label)
+        self.key = (self.start, self.label)
```

Comment:
An `Edge` is basically like a struct. All member are public and likely being used by user land code..